containers/ws/README.md

```diff
@@ -119,6 +119,10 @@ You can also mount encrypted private keys inside the container. You can set an e
 
 Private keys can be encrypted; then cockpit uses the provided password to decrypt the key.
 
+### SSH_AUTH_SOCK
+
+By default the container starts its own ssh-agent. Alternatively `SSH_AUTH_SOCK` environment variable can be set to surpress starting ssh-agent.
```

Comment:
"surpress" typo, and I think it would be better to describe a "positive" use case:
```suggestion
By default the container starts its own ssh-agent. Alternatively you can pass an `SSH_AUTH_SOCK` environment variable and a corresponding socket volume to the container to re-use your host's socket.
```

and perhaps give an example? Does this work?
```
-v $SSH_AUTH_SOCK:/ssh.sock -e SSH_AUTH_SOCK=/ssh.sock
```

Reply:
This might under a non SElinux system, for now I'm just running it as `--privileged`.

Otherwise, yeah, that works.

Reply:
Ah, perhaps with `-v $SSH_AUTH_SOCK:/ssh.sock:Z` then?